src/utils/list.js

```diff
@@ -0,0 +1,38 @@
+import R from 'ramda';
+import natsort from 'natsort';
+
+const sortObjects = natsort();
+
+/**
+ * Sort an array of objects by property.
+ *
+ * @param {String|Array} prop
+ * A property to sort by. If passed as an array, it
+ * will be treated as a deep property path.
```

Comment:
BTW, I like how you used the line below the `@param` to add a description. Feels better than if it all happened on the same line. :)